pkgs/racket-doc/scribblings/guide/other-editors.scrbl

```diff
@@ -248,11 +228,8 @@ support Racket out of the box: @itemlist[
 
 @subsection{Scribble}
 
-Vim support for writing scribble documents is provided by @itemlist[
-    @item{@hyperlink["https://github.com/wilbowma/scribble.vim"]{wilbowma/scribble.vim}}
-    @item{@hyperlink["https://github.com/benknoble/scribble.vim"]{benknoble/scribble.vim}}
-]@margin-note{Again, @tt{benknoble/scribble.vim} is updated more frequently and
-is somewhat more modern.}
+Vim support for writing scribble documents is provided by
+@hyperlink["https://github.com/benknoble/scribble.vim"]{benknoble/scribble.vim}.
```

Comment:
Outside the scope of this PR, but I'd consider updating the README to be more current and explain the provenance of the repo in an acknowledgements section.